app/api/cruds/utils.py

```diff
@@ -0,0 +1,19 @@
+from django.contrib.auth.models import User
+from graphql_relay.node.node import from_global_id
+
+
+def get_errors(e):
+    # transform django errors to redux errors
+    # django: {"key1": [value1], {"key2": [value2]}}
+    # redux: ["key1", "value1", "key2", "value2"]
+    fields = e.message_dict.keys()
+    messages = ['; '.join(m) for m in e.message_dict.values()]
+    errors = [i for pair in zip(fields, messages) for i in pair]
+    return errors
+
```

Comment:
I think a single blank line is sufficient here; it would have been necessary if they were classes.